### ISSUE TITLE
Clean up translations files

### ISSUE THREAD

**Issue by lennerd** [sozialhelden/wheelmap#78]
At the moment we have eight translation files. Adding new languages is a huge amount of work and cost a lot of money. At the same time there are many legacy translations keys not used any more or used in old parts of the wheelmap.

So we have an urgent need to clean up our translations files (at best to have only two or three well organized single files).

**Comment by holgerd** [comment]
Did you discuss this with Silke already? She has done a lot of cleanup work already afaik.

**Comment by lennerd** [comment]
Yes, Jonas, Silke and I talked about it. They fixed some source language strings but that didn't clean up the Transifex project. Unfortunately this needs to be done by someone, who can read code to see if the translation keys are really used.

**Comment by 1000miles** [comment]
Maybe related:

Github issue: https://github.com/sozialhelden/wheelmap/issues/568
PR: https://github.com/sozialhelden/wheelmap/pull/576
